src/main/java/com/skytask/repository/ProductRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.skytask.repository;
+
+import com.skytask.model.Product;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.query.Procedure;
+
+public interface ProductRepository extends JpaRepository<Product, Long> {
+
+    @Procedure
+    int insert_product(String name, String description, Double price, Integer stock);
```

Comment:
Where is the store procedure definition? could you include it please in this repo? (it is not needed but if you have time it is always nice to use features like FlyWay or liquibase, IMO it is a lot easier FlyWay)